examples/3p.error.amp.html

```diff
@@ -0,0 +1,1436 @@
+<!doctype html>
```

Comment:
why is this file? 
for a testing example, you can put in `analytics-user-error.amp.html` once that PR is merged

Reply:
+1